packages/react-devtools-shared/src/utils.js

```diff
@@ -78,7 +78,7 @@ export function getAllEnumerableKeys(
     const descriptors = Object.getOwnPropertyDescriptors(current);
     currentKeys.forEach(key => {
       // $FlowFixMe: key can be a Symbol https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor
-      if (descriptors[key].enumerable) {
+      if (descriptors[key] && descriptors[key].enumerable) {
```

Comment:
Do you have an example of where this would have thrown before? (Can we add a test for it to ensure that we don't regress?)